app/common/controllers/intercom.js

```diff
@@ -0,0 +1,28 @@
+module.exports = [
+    '$scope',
+    '$rootScope',
+    'UserEndpoint',
+function (
+    $scope,
+    $rootScope,
+    UserEndpoint
+) {
+    $rootScope.$on('event:authentication:login:succeeded', function () {
+        UserEndpoint.getFresh({id: 'me'}).$promise.then(function (user) {
+            window.Intercom('boot', {
+                app_id: window.ushahidi.intercomAppId,
+                email: user.email,
+                created_at: user.created,
+                user_id: user.id,
```

Comment:
We should have the domain/subdomain for the user so we know what deployment they come from.